src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -39,12 +39,14 @@ export class RundownTimelineService implements RundownService {
   public async activateRundown(rundownId: string): Promise<void> {
     await this.assertNoRundownIsActive()
     const rundown: Rundown = await this.rundownRepository.getRundown(rundownId)
-
+    const infinitePiecesBeforeActivation: Map<string,Piece> = rundown.getInfinitePiecesMap()
     rundown.activate()
 
     await this.buildAndPersistTimeline(rundown)
-
-    this.rundownEventEmitter.emitInfinitePiecesUpdatedEvent(rundown)
+    const infinitePiecesAfterActivation: Map<string,Piece> = rundown.getInfinitePiecesMap()
```

Comment:
Minor thing, but it would be nice to have a space after the comma and before the Piece in `Map<string,Piece>`, so it becomes `Map<string, Piece>`.
It makes it easier to read when everything isn't cramped :)